src/modules/components/layout/masthead-link-box/masthead-link-box.scss

```diff
@@ -0,0 +1,8 @@
+$module: 'masthead-link-box';
+
+.#{$module} {
+  height: 215px;
```

Comment:
Not a big fan of this as it will break if the left side content is taller than the right 🤔

Reply:
Yeah, also, when the content wraps underneath on a responsive display, you will likely have an un-necessarily large height because it's fixed at 215px.

Also I am not sure that this box can ever be said to be  215px in height, because it would surely depend on both the content (i.e. number of links) and the width of the viewport (what happens if a link were to wrap, for example?).

Ultimately, the best solution here would be to re-model the pdk-masthead to use flex or table display, to solve this perfectly, but that is more of a comprehensive change. So, perhaps the best comprimise might be to use something like this:

```
@include media(tablet) {
  min-height: 215px;
}
```
As for integration: once commited, you'll need to get someone to publish a new version of PDK, which you can then update to in your app. Fabio is probably the most available to do this.